core/src/banking_stage.rs

```diff
@@ -58,14 +60,9 @@ mod multi_iterator_scanner;
 mod packet_deserializer;
 mod packet_receiver;
 mod read_write_account_set;
-
 #[allow(dead_code)]
 mod scheduler_messages;
-
-mod consume_worker;
-mod forward_worker;
-#[allow(dead_code)]
-mod thread_aware_account_locks;
+mod transaction_scheduler;
```

Comment:
@ryoqun thought you might have some naming thoughts here.

IMO it's sufficiently nested within banking stage that it's fine, but wanted to give you a prompted chance to refute 😄